src/components/pages/dashboard/dashboard.js

```diff
@@ -124,7 +124,7 @@ class DashboardPage extends Component {
           <Grid fluid className="layout">
             <Row>
               <Col md={8}>
-                <DeviceMap {...deviceMapProps} />
+                {this.props.BingMapKey ? <DeviceMap {...deviceMapProps}/> : null} 
```

Comment:
could use { this.props.BingMapKey && <DeviceMap {...deviceMapProps}/> }

Reply:
Both are same.